src/windows/osd.c

```diff
@@ -56,36 +56,35 @@ void fi_fini(void);
 
 int socketpair(int af, int type, int protocol, int socks[2])
 {
-	protocol; /* suppress warning */
+	OFI_UNUSED(protocol);
+
 	struct sockaddr_in in_addr;
 	int lsock;
 	int len = sizeof(in_addr);
 
-	if(!socks)
-	{
+	if(!socks) {
 		WSASetLastError(WSAEINVAL);
 		return SOCKET_ERROR;
 	}
 
 	socks[0] = socks[1] = (int)INVALID_SOCKET;
-	if ((lsock = socket(af == AF_UNIX ? AF_INET : af, type, 0)) == INVALID_SOCKET)
-	{
+	if ((lsock = socket(af == AF_UNIX ? AF_INET :
```

Comment:
To me, the line break between `AF_INET :` and `af` makes the code difficult to read.

Reply:
I've tried to make length of line a bit shorter to get into 80 characters per line. Will update to make this more readable